### PR TITLE
fix(#minor); gmx; fix position snapshots

### PR DESCRIPTION
**Pull request by dhruv-chauhan**
Deployments,

avalanche: https://okgraph.xyz/?q=Qmemei77paarcRzt1exCzcNuBJhjTmeRsLHSAxDGTLjfxR
arbitrum: https://okgraph.xyz/?q=Qmf1vnAzQS4SCgy3nzEy1m1X5iGgbqMobGPJtKU8E4o7v3

Example query,

https://api.thegraph.com/subgraphs/id/Qmemei77paarcRzt1exCzcNuBJhjTmeRsLHSAxDGTLjfxR/graphql?query=%7B%0A++positions+%7B+%0A++++id%0A++++timestampOpened%0A++++hashOpened%0A++++timestampClosed%0A++++hashClosed%0A++++snapshots%28orderBy%3A+timestamp%2C+orderDirection%3A+asc%29+%7B+%0A++++++balance+%0A++++++balanceUSD+%0A++++++collateralBalance+%0A++++++collateralBalanceUSD+%0A++++++timestamp+%0A++++++hash%0A++++%7D+%0A++%7D%0A%7D

### CODE REVIEW

**Comment by ishraq8** [comment]
passed QA

File: subgraphs/gmx-forks/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "@graphprotocol/graph-ts": "0.29.1"
   },
   "devDependencies": {
-    "matchstick-as": "0.5.0"
+    "matchstick-as": "0.5.0",
```
Comment (dmelotik) [inline subgraphs/gmx-forks/package.json:12]: let's remove this since it is not being used

**Review by dmelotik — COMMENTED** [review]
Looks good! Just 1 extra comment, can you copy all of the sdk changes over to the gmx-forks repo so we don't have some files that show the old version.

Thanks!

**Comment by ishraq8** [comment]
Following discussion here: https://discord.com/channels/953684103012683796/1067631209968058408/1126475904944705626

do you think you can add `inputTokens` into the daily and hourly pool level time series snapshot with `inputTokenBalances` and `inputTokenWeights` in the schema and sdk.

Also update the GMX updates you have with this? 

If not, no worries, i can add it after this is merged.

**Comment by dhruv-chauhan** [comment]
@ishraq8 added, and updated deployment links in the description

File: subgraphs/gmx-forks/yarn.lock
Comment (dmelotik) [inline subgraphs/gmx-forks/yarn.lock:1]: we don't rly need a `package-lock.json` and a `yarn.lock`, but I am still going to merge with this

**Comment by ishraq8** [comment]
updates to `inputtokens` appears to be working, thank you @dhruv-chauhan

**Comment by dmelotik** [comment]
@dhruv-chauhan can we also update versions for `mummy-finance`

**Comment by dhruv-chauhan** [comment]
@dmelotik done

**Review by dmelotik — APPROVED** [review]

**Pull request merged**